ha/setup/ha_setup.py

```diff
@@ -344,6 +344,17 @@ def process(self):
             node_id = Conf.get(self._index, f"server_node.{machine_id}.node_id")
             self._confstore.set(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}", node_id)
 
+        node_id = Conf.get(self._index, f"server_node.{machine_id}.node_id")
+        cluster_id = Conf.get(self._index, f"server_node.{machine_id}.cluster_id")
+        site_id = Conf.get(self._index, f"server_node.{machine_id}.site_id")
+        rack_id = Conf.get(self._index, f"server_node.{machine_id}.rack_id")
+        storageset_id = Conf.get(self._index, f"server_node.{machine_id}.storageset_id")
+
+        # Update node map
+        node_map = {'cluster_id': cluster_id, 'site_id': site_id,
```

Comment:
Above we have added a new file and class for these strings e.g. 'cluster_id'. Lets use that here instead of the hard-coded strings.

Reply:
Done.